third_party/2and3/redis/client.pyi

```diff
@@ -221,7 +221,7 @@ class Redis(object):
     def smembers(self, name: _Key) -> Set[_Value]: ...
     def smove(self, src, dst, value): ...
     def spop(self, name, count: Optional[int] = ...): ...
-    def srandmember(self, name, number=...): ...
+    def srandmember(self, name: _Key, number: Optional[int] = ...) -> _Value: ...
```

Comment:
This returns `Optional[_Value]` if `number` is not given or is `None` and a list of values if `number` is not `None`. We should use overloads to reflect this.